docs/rancher/cloud-provider.md

```diff
@@ -59,6 +59,11 @@ When spinning up an RKE2 cluster using the Harvester node driver, select the `Ha
 
   ![](/img/v1.2/rancher/rke2-cloud-provider.png)
 
+With Rancher version above 2.9.0, to customize folder for the cloud config data, you can modify `Data directory configuration path`.
```

Comment:
```suggestion
Starting with Rancher v2.9.0, you can configure a specific folder for cloud config data using the **Data directory configuration path** field.
```

Reply:
We use backticks for code and bold for most GUI elements.

Reply:
Thanks for the review. Updated it.